server/src/main/java/org/activityinfo/server/endpoint/akvo/flow/AkvoFlowResource.java

```diff
@@ -0,0 +1,155 @@
+package org.activityinfo.server.endpoint.akvo.flow;
+
+import com.google.inject.Inject;
+import liquibase.util.csv.CSVReader;
+import org.activityinfo.legacy.shared.command.CreateEntity;
+import org.activityinfo.legacy.shared.command.GetSchema;
+import org.activityinfo.legacy.shared.command.UpdateFormClass;
+import org.activityinfo.legacy.shared.command.result.CreateResult;
+import org.activityinfo.legacy.shared.model.ActivityFormDTO;
+import org.activityinfo.legacy.shared.model.LocationTypeDTO;
+import org.activityinfo.legacy.shared.model.UserDatabaseDTO;
+import org.activityinfo.model.form.FormClass;
+import org.activityinfo.model.form.FormField;
+import org.activityinfo.model.form.FormInstance;
+import org.activityinfo.model.legacy.CuidAdapter;
+import org.activityinfo.model.resource.ResourceId;
+import org.activityinfo.model.resource.Resources;
+import org.activityinfo.model.type.enumerated.EnumItem;
+import org.activityinfo.model.type.enumerated.EnumType;
+import org.activityinfo.server.command.DispatcherSync;
+import org.activityinfo.server.command.ResourceLocatorSync;
+import org.activityinfo.server.endpoint.rest.RootResource;
+import org.activityinfo.service.feed.FeedService;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.FormParam;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriInfo;
+import java.io.IOException;
+import java.io.StringReader;
+import java.util.logging.Logger;
+
+import static java.util.logging.Level.SEVERE;
+import static javax.ws.rs.core.MediaType.APPLICATION_FORM_URLENCODED;
+import static javax.ws.rs.core.MediaType.TEXT_PLAIN;
+import static javax.ws.rs.core.Response.Status.BAD_REQUEST;
+import static javax.ws.rs.core.Response.Status.UNAUTHORIZED;
+
+@Path("/AkvoFlow")
```

Comment:
Please make a subpath of "/tasks/" which has the right permissions defined in web.xml so that it can not be triggered by normal users.